### PR TITLE
feature(*): support exec subcommand

### PR DESCRIPTION
**Pull request by aspirer**
issue #153
usage:
        curveadm exec 0e5d809f16b4 ps
        curveadm exec 0e5d809f16b4 curve_ops_tool status
        curveadm exec 0e5d809f16b4 -- ps -ef

Signed-off-by: Wangpan <aspirer2004@gmail.com>

```
netease@hih-l-9493:~/curveadm$ ./bin/curveadm exec -h
Usage:  curveadm exec ID [OPTIONS]

Exec a cmd in service container

Options:
  -h, --help   Print usage

netease@hih-l-9493:~/curveadm$ ./bin/curveadm exec 0e5d809f16b4 
Connection to 127.0.0.1 closed.
netease@hih-l-9493:~/curveadm$ ./bin/curveadm exec 0e5d809f16b4 -h
Usage:  curveadm exec ID [OPTIONS]

Exec a cmd in service container

Options:
  -h, --help   Print usage

netease@hih-l-9493:~/curveadm$ ./bin/curveadm exec 0e5d809f16b4 ps
    PID TTY          TIME CMD
   1652 pts/3    00:00:00 ps
Connection to 127.0.0.1 closed.

netease@hih-l-9493:~/curveadm$ ./bin/curveadm exec 0e5d809f16b4 "ps -ef"
UID          PID    PPID  C STIME TTY          TIME CMD
root           1       0  0 Nov17 ?        00:00:01 /sbin/docker-init -- /entrypoint.sh --role mds --args=

netease@hih-l-9493:~/curveadm$ ./bin/curveadm exec 0e5d809f16b4 -- ps -ef
UID          PID    PPID  C STIME TTY          TIME CMD
root           1       0  0 Nov17 ?        00:00:01 /sbin/docker-init -- /entrypoint.sh --role mds --args=

netease@hih-l-9493:~/curveadm$ ./bin/curveadm exec 0e5d809f16b4 curve_ops_tool status | head -n3
Cluster status:
Send RPC to chunkserver fail, error content: [E112]Not connected to 127.0.0.1:8200 yet, server_id=6 [R1][E112]Not connected to 127.0.0.1:8200 yet, server_id=6 [R2][E112]Not connected to 127.0.0.1:8200 yet, server_id=6 [R3][E112]Not connected to 127.0.0.1:8200 yet, server_id=6
Send RPC to chunkserver fail, error content: [E112]Not connected to 127.0.0.1:8202 yet, server_id=8 [R1][E112]Not connected to 127.0.0.1:8202 yet, server_id=8 [R2][E112]Not connected to 127.0.0.1:8202 yet, server_id=8 [R3][E112]Not connected to 127.0.0.1:8202 yet, server_id=8
Connection to 127.0.0.1 closed.
netease@hih-l-9493:~/curveadm$ ./bin/curveadm exec 0e5d809f16b4 -- curve_ops_tool status | head -n3
Cluster status:


```

### CODE REVIEW

**Comment by Cyber-SiKu** [comment]
Is `curveadm exec [--id xxxx] cmd` better? The id can be omitted and it will be executed on the main mds by default.

File: cli/command/exec.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveAdm
+ * Created Date: 2021-10-15
+ * Author: Jingli Chen (Wine93)
```
Comment (Cyber-SiKu) [inline cli/command/exec.go:20]: name

File: cli/command/exec.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+ */
+
+// __SIGN_BY_WINE93__
```
Comment (Cyber-SiKu) [inline cli/command/exec.go:23]: ditto

File: cli/command/exec.go [hunk 3]
```diff
@@ -0,0 +1,92 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
```
Comment (Cyber-SiKu) [inline cli/command/exec.go:2]: 2022

**Comment by Wine93** [comment]
How about omit the explicit `cmd` argument, like:

```
curveadm exec 0e5d809f16b4 ps -ef
```

it's more like docker's `exec` command:

```
docker exec 4ac34856f43f ps -ef
```

**Comment by aspirer** [comment]
> How about omit the explicit `cmd` argument, like:
> 
> ```
> curveadm exec 0e5d809f16b4 ps -ef
> ```
> 
> it's more like docker's `exec` command:
> 
> ```
> docker exec 4ac34856f43f ps -ef
> ```

usage:
curveadm exec 0e5d809f16b4 ps
curveadm exec 0e5d809f16b4 curve_ops_tool status
curveadm exec 0e5d809f16b4 -- ps -ef

**Pull request merged**